spec/index.html

```diff
@@ -1174,23 +1174,67 @@ <h5>DELETE WHERE</h5>
-          a transmission error where partial data has been received (which itself may be legal RDF).</p>
+
+          <p>
+            The `LOAD` operation reads an RDF document from a IRI and inserts triples
+            into the graphs of the dataset in the  <a href="#defn_graphStore">Graph Store</a>.
+          </p>
+
+          <pre class="defn nohighlight"><strong>LOAD</strong> ( <strong>SILENT</strong> )? <em><a
+            data-cite="SPARQL12-QUERY#rIRIREF">IRIref_from</a></em> ( <strong>INTO GRAPH</strong> <em><a
+            data-cite="SPARQL12-QUERY#rIRIREF">IRIref_to</a></em> )?</pre>
+
+          <p><em>IRIref_from</em> specifies the IRI of a document such that a store will be able to
+            identify, locate and read the document. The most common form of IRIs will be URLs with
+            the https and http IRI schemes.
+          </p>
+
+          <p>If a destination graph (<i>IRIref_to</i>) is specified by `INTO GRAPH`, then the
+            specified destination graph SHOULD be created if required; again, implementations
+            providing an update service over a fixed set of graphs MUST return with failure for a
+            request that would create a disallowed graph. If the destination graph already exists,
+            then no data in that graph will be removed. 
+            The RDF document is retrieved and parsed to produce an
+            <a data-cite="RDF12-CONCEPTS#dfn-rdf-graph">RDF Graph</a>.
+            This graph is
+            <a data-cite="RDF12-SEMANTICS#dfn-merging">merged</a> into the destination
+            graph in the graph store..
+          </p>
+          <p>
+            If no destination graph IRI is specified by `INTO`, then the RDF document is retrived
+            and parsed into an <a data-cite="RDF12-CONCEPTS#dfn-rdf-dataset">RDF Dataset</a>. 
+            This dataset is <a data-cite="RDF12-SEMANTICS#dfn-rdf-dataset-merge">merged</a> into the
+            <a href="#defn_graphStore">graph store</a> viewed as a mutable
+            <a data-cite="SPARQL12-QUERY#sparqlDataset">RDF Dataset</a>.
+          </p>
+
+          <p>
+            In the case where no RDF data can be retrieved (as opposed to the empty graph being retrieved)
+            from the IRI denoted by <em>IRIref_from</em>, or in case the retrieval method returns an
+            error (such as, for instance an HTTP error code), the SPARQL 1.2 Update service SHOULD
+            return failure and the status of the Graph Store SHOULD remain in the same status as
+            prior to the request; in case the keyword `SILENT` is present, however, the
+            operation will still return success and the status of the Graph Store is not specified
+            by the present document: implementations may create the destination graph or not and
+            partially load data, in case of a transmission error where partial data has been
+            received (which itself may be legal RDF).
```

Comment:
```suggestion
            If no RDF data can be retrieved (as opposed to an empty graph being retrieved)
            from the IRI denoted by <em>IRIref_from</em>, or if the retrieval method returns an
            error (such as, for instance an HTTP error code), the SPARQL 1.2 Update service SHOULD
            return failure, and the status of the Graph Store SHOULD remain in the same status as
            prior to the request; if the keyword `SILENT` is present, however, the
            operation will return success, and the status of the Graph Store is not specified
            by the present document; i.e., implementations might — or might not — create the destination graph and
            might load partial data, such as when a transmission error causes partial data to be
            received (which itself may be legal RDF).
```

Reply:
This is original text. Fixes applied for the first part applied.

The description of only some possible out-of-spec states in this one operation does not seem helpful. Sentence removed.